ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -113,6 +113,55 @@
   @Default("true")
   public final boolean attachTrackingInfo;
 
+  /**
+   * Max QPS allowed for http post. Set to -1 if there is no QPS limitation.
+   */
+  @Config("frontend.http.post.max.qps")
```

Comment:
instead of multiple configs, I think this should be defined as a json/map. Having/setting a config for each request type is too much overhead

Reply:
An individual json file on disk?

Reply:
That's a good point because we may need to set quotas for many containers in the future.

Reply:
A json file in config for now, we can look at better options when we plan to throttle by container